lib/repo_tools.pm

```diff
@@ -132,6 +132,9 @@ sub prepare_source_repo {
         elsif (is_sle('>=12-SP4') and get_var('REPO_SLES_SOURCE')) {
             zypper_call("ar -f " . "$utils::OPENQA_FTP_URL/" . get_var('REPO_SLES_SOURCE') . " repo-source");
         }
+        elsif (is_sle('>=12-SP4') and get_var('REPO_SLES_POOL_SOURCE')) {
+            zypper_call("ar -f " . "$utils::OPENQA_FTP_URL/" . get_var('REPO_SLES_POOL_SOURCE') . " repo-source");
```

Comment:
What about something like this to avoid another elsif?:
`my $repo_source = get_var('REPO_SLES_SOURCE',  get_var('REPO_SLES_POOL_SOURCE'));
zypper_call("ar -f " . "$utils::OPENQA_FTP_URL/" . $repo_source . " repo-source") if $repo_source;`

Reply:
That would prevent us from getting into the remaining `elsif` / `else` blocks if we wouldn't check for `REPO_SLES_SOURCE` *and* `REPO_SLES_POOL_SOURCE` in our `elsif`.

Reply:
if you wrap the previous code with `elsif (is_sle('>=12-SP4')` and continue with rest of elsif/else I think is the same flow.

Reply:
I am missing the ` if $repo_source;` in your last snippet

Reply:
wait, got it! sorry

Reply:
I guess I was thinking in something like this:
```
if (is_sle) {
        my $repo_source = get_var('REPO_SLES_SOURCE', 'REPO_SLES_POOL_SOURCE');
        if (is_sle('>=15') and get_var(get_repo_var_name("MODULE_BASESYSTEM_SOURCE"))) {
            zypper_call("ar -f " . "$utils::OPENQA_FTP_URL/" . get_var(get_repo_var_name("MODULE_BASESYSTEM_SOURCE")) . " repo-source");
        }
        elsif (is_sle('>=12-SP4') and $repo_source) {
          zypper_call("ar -f " . "$utils::OPENQA_FTP_URL/" . $repo_source . " repo-source");
        }
```

Reply:
well, let's go ahead as is. your change could work though. Feel free to propose another PR so that we can even easily see the difference to the current, new version more easily.